packages/maskbook/src/plugin-infra/register.ts

```diff
@@ -5,5 +5,6 @@ import '@dimensiondev/plugin-example'
 import '../plugins/RedPacket'
 import '../plugins/Snapshot'
 import '../plugins/Collectible'
-// import '../plugins/NFT'
 import '../plugins/External'
+// import '../plugins/NFT'
+// import '../plugins/Airdrop'
```

Comment:
It registered before, but not now. Is that intended?

Reply:
Yes. They didn't be used before.